src/main/resources/templates/db_scripts/instanceDeleteCascade.sql

```diff
@@ -0,0 +1,10 @@
+-- When deleting an instance automatically delete the connected marc record.
+-- Approach: foreign key "ON DELETE CASCADE"
+
+ALTER TABLE ${myuniversity}_${mymodule}.instance_source_marc
+  DROP CONSTRAINT IF EXISTS instance_source_marc_id_fkey;
+
+ALTER TABLE ${myuniversity}_${mymodule}.instance_source_marc
+  ADD CONSTRAINT instance_source_marc_id_fkey
+  FOREIGN KEY (id) REFERENCES instance(id) ON DELETE CASCADE;
```

Comment:
I think it's interesting that active changes are being made in this area.

No one should be using MARC records stored directly in inventory and this API should've been removed a long time ago.

How has it been decided to cascade the deletion rather than fail deletion when a MARC record is present?

Reply:
You should have asked this question before you merged the pull request four years ago:
https://github.com/folio-org/mod-inventory-storage/pull/126/files#diff-dbf6259d56a78ce52a3e91870b73432e7ac83f8b06f58a22135feb02fb17a9bdR349

Reply:
I couldn't find a Jira that approves the removal of the MARC records APIs.

Reply:
There may well not be one. 

I would hope it was obvious that it has been superseded by mod-source-record-storage.

I thought those APIs were informally deprecated a long time ago.